enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -19,6 +20,8 @@
 from ..processproxies.processproxy import LocalProcessProxy, RemoteProcessProxy
 from ..sessions.kernelsessionmanager import KernelSessionManager
 
+UNIVERSAL_TENANT_ID: str = "27182818-2845-9045-2353-602874713527"
```

Comment:
The UUID should be a static value because it will be persisted with the kernel session information.  So, when persistence is enabled, all EG servers should have the same UNIVERSAL_TENANT_ID for backwards compatibility.